lib/tool_shed/scripts/api/common.py

```diff
@@ -4,12 +4,9 @@
 import urllib
 import urllib2
 
-new_path = [ os.path.join( os.path.dirname( __file__ ), os.pardir, os.pardir, os.pardir, os.pardir, 'lib' ) ]
-new_path.extend( sys.path[ 1: ] )
-sys.path = new_path
+sys.path.insert(1, os.path.join( os.path.dirname( __file__ ), os.pardir, os.pardir, os.pardir ) )
```

Comment:
No, this file is already in a subdirectory of `lib`, so I have removed `os.pardir, 'lib'` .